packages/web-components/fast-components/src/accordion/accordion.styles.ts

```diff
@@ -1,19 +1,23 @@
 import { css } from "@microsoft/fast-element";
 import { display } from "@microsoft/fast-foundation";
 import {
-    neutralDividerRestBehavior,
-    neutralForegroundRestBehavior,
-} from "../styles/recipes";
+    bodyFont,
+    neutralForegroundRest,
+    neutralStrokeDividerRest,
+    strokeWidth,
+    typeRampMinus1FontSize,
+    typeRampMinus1LineHeight,
+} from "../design-tokens";
 
-export const AccordionStyles = css`
-    ${display("flex")} :host {
-        box-sizing: border-box;
-        flex-direction: column;
-        font-family: var(--body-font);
-        font-size: var(--type-ramp-minus-1-font-size);
-        line-height: var(--type-ramp-minus-1-line-height);
-        color: ${neutralForegroundRestBehavior.var};
-        border-top: calc(var(--outline-width) * 1px) solid
-            ${neutralDividerRestBehavior.var};
-    }
-`.withBehaviors(neutralDividerRestBehavior, neutralForegroundRestBehavior);
+export const accordionStyles = (context, definition) =>
```

Comment:
We should *probably* explicitly type `context` and `definition`

Reply:
I can probably pick this up with a PR if you want to pick up the exports and the versions?

Reply:
Resolved and in the commit history now :)